README.md

```diff
@@ -1,156 +1,33 @@
-![CI tests](https://github.com/PEtab-dev/PEtab/workflows/CI%20tests/badge.svg)
-[![Codacy Badge](https://api.codacy.com/project/badge/Grade/fd7dd5cee68e449983be5c43f230c7f3)](https://www.codacy.com/gh/PEtab-dev/PEtab?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=PEtab-dev/PEtab&amp;utm_campaign=Badge_Grade)
-[![codecov](https://codecov.io/gh/PEtab-dev/PEtab/branch/master/graph/badge.svg)](https://codecov.io/gh/PEtab-dev/PEtab)
+![CI tests](https://github.com/PEtab-dev/libpetab-python/workflows/CI%20tests/badge.svg)
+[![Codacy Badge](https://api.codacy.com/project/badge/Grade/fd7dd5cee68e449983be5c43f230c7f3)](https://www.codacy.com/gh/PEtab-dev/libpetab-python)
+[![codecov](https://codecov.io/gh/PEtab-dev/libpetab-python/branch/master/graph/badge.svg)](https://codecov.io/gh/PEtab-dev/libpetab-python)
 [![PyPI version](https://badge.fury.io/py/petab.svg)](https://badge.fury.io/py/petab)
 
-# PEtab -- a data format for specifying parameter estimation problems in systems biology
+# `petab` - a Python package for handling PEtab files
```

Comment:
```suggestion
# petab - a Python package for handling PEtab files
```
Looks ugly imho otherwise. Do we lowercase peTAB now always?

Reply:
"PEtab" when referring to the format, "petab" when referring to the Python package, not?

Reply:
:+1: